uxarray/cross_sections/dataarray_accessor.py

```diff
@@ -17,60 +17,84 @@ def __repr__(self):
         prefix = "<uxarray.UxDataArray.cross_section>\n"
         methods_heading = "Supported Methods:\n"
 
-        methods_heading += "  * constant_latitude(center_coord, k, element, **kwargs)\n"
+        methods_heading += "  * constant_latitude(lat, use_spherical_bounding_box)\n"
+        methods_heading += "  * constant_longitude(lon, use_spherical_bounding_box)\n"
 
         return prefix + methods_heading
 
-    def constant_latitude(self, lat: float, use_spherical_bounding_box=False):
-        """Extracts a cross-section of the data array at a specified constant
-        latitude.
+    def constant_latitude(self, lat: float, use_spherical_bounding_box: bool = True):
+        """Extracts a cross-section of the data array by selecting all faces that
+        intersect with a specified line of constant latitude.
 
         Parameters
         ----------
         lat : float
             The latitude at which to extract the cross-section, in degrees.
+            Must be between -90.0 and 90.0
         use_spherical_bounding_box : bool, optional
-            If True, uses a spherical bounding box for intersection calculations.
+            If True, uses a spherical bounding box for obtaining candidate faces, which considers the extreme cases
+            along great circle arcs.
+
+        Returns
+        -------
+        uxarray.UxDataArray
+            A subset of the original data array containing only the faces that intersect
+            with the specified latitude.
 
         Raises
         ------
         ValueError
-            If no intersections are found at the specified latitude, a ValueError is raised.
+            If no intersections are found at the specified latitude.
 
         Examples
         --------
-        >>> uxda.constant_latitude_cross_section(lat=-15.5)
+        >>> # Extract data at 15.5°S latitude
+        >>> cross_section = uxda.constant_latitude(lat=-15.5)
+
+        Notes
+        -----
+        The initial execution time may be significantly longer than subsequent runs
+        due to Numba's just-in-time compilation. Subsequent calls will be faster due to caching.
         """
         faces = self.uxda.uxgrid.get_faces_at_constant_latitude(
             lat, use_spherical_bounding_box
         )
 
         return self.uxda.isel(n_face=faces)
 
-    def constant_longitude(self, lon: float, use_spherical_bounding_box=False):
-        """Extracts a cross-section of the data array at a specified constant
-        longitude.
+    def constant_longitude(self, lon: float, use_spherical_bounding_box: bool = True):
```

Comment:
since the longnitue will not suffer from the curvature, we don't actually need to build the bounding box here.

Reply:
Good point, although the alternative if we didn't use the bounding box would require us to check the intersection on every edge of every face, which is more expensive that checking the minimum and maximum latitude for each face when. Using the bounding box should be the preferred method if many cross section operations are being performed. 

I will correct the docstrings regarding the extremes though.

Reply:
If the lat-lon bounds aren’t pre-built, calling this function will end up looping through each edge of the faces and performing extra latitude checks, which can be pretty slow.

Ideally, if the lat-lon bounds are already available, we can use them directly. If not, we could do a quick longitude overlap check to keep things efficient.

For now, let’s just add a note about this in the docstring and leave it as a potential improvement for the future